azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/ContainerService.java

```diff
@@ -0,0 +1,275 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.compute;
+
+import com.microsoft.azure.management.apigeneration.Beta;
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.compute.implementation.ComputeManager;
+import com.microsoft.azure.management.compute.implementation.ContainerServiceInner;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.GroupableResource;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.Resource;
+import com.microsoft.azure.management.resources.fluentcore.model.Appliable;
+import com.microsoft.azure.management.resources.fluentcore.model.Creatable;
+import com.microsoft.azure.management.resources.fluentcore.model.Refreshable;
+import com.microsoft.azure.management.resources.fluentcore.model.Updatable;
+
+/**
+ * An client-side representation for a container service.
+ */
+@Fluent
+@Beta
+public interface ContainerService extends
+        GroupableResource<ComputeManager, ContainerServiceInner>,
+        Refreshable<ContainerService>,
+        Updatable<ContainerService.Update> {
+
+    /**
+     * @return the properties of the orchestrator
+     */
+    ContainerServiceOrchestratorProfile orchestratorProfile();

```

Comment:
note for the future (not required for the initial PR) --- we'll probably want to flatten at least some of these "*Profile" thingies for better usability/discoverability of what matters.